src/main/java/org/fundacion/jala/converter/view/controllers/ClientRequest.java

```diff
@@ -35,7 +35,6 @@
 import java.net.MalformedURLException;
 import java.net.URL;
 import static org.fundacion.jala.converter.ConverterApplication.dotenv;
-import static org.fundacion.jala.converter.models.UserSQL.findUserById;
 
 public class ClientRequest {

```

Comment:
Is missing javaDoc of the class ClientRequest

Reply:
added.